### DISCUSSION TITLE
Is the Notion API blocked on Vercel or vice versa?

### DISCUSSION THREAD

**Discussion by SSardorf** [vercel/community#371] [Help]
I have an API that adds customer data to our CRM system and also some information to our Notion board. 

It involves three external API calls.
Two to our CRM system, and one to our Notion board.

The first two work flawlessly. However, when it gets to the Notion API call, the call just hangs when deployed to production. Locally it works fine.

I have also made sure that the environment variables are identical locally and in production

Here's the codeblock
```js
console.log("Notion api should trigger now")
            axios({
                method: 'POST',
                url: 'https://api.notion.com/v1/pages/',
                headers: {
                    Authorization: `Bearer ${notionKey}`,
                    'Content-Type': 'application/json',
                    'Notion-Version': '2021-05-13',
                },
                timeout: 10000,
                data: data,
            }).then((response)=>{
                console.log(response)
            }).catch((error) => {
                console.log(error);
                postToSlack({ error_loc: 'notionLead', data: data });
                postToSlack({
                    error_msg: error.response.data,
                });
            });
```

The only log I get from Vercel is :
[![enter image description here][1]][1]

Link to the whole file

https://gist.github.com/SSardorf/e0421d04d1f9857cb3f2d934d92a5e9f#file-ac_contact-js-L76-L86

  [1]: https://i.stack.imgur.com/s4CLM.png

**Comment by leerob** [comment]
No blocks on our side, we have other demos using the Notion API that are working as expected 👍

**Reply by SSardorf** [reply]
Do you also have demos using notion POST endpoints? Could only find GET endpoints. Super confused as to why what I am doing is not working

**Comment by SSardorf** [comment]
Actually, now I am getting this, four minutes after the post request.

<img width="1059" alt="image" src="https://user-images.githubusercontent.com/22572423/154704822-b29e488b-5907-4328-8989-399e028127d7.png">

**Reply by samcx** [reply]
@SSardorf Apologies for the late response! Are you still experiencing an issue?

**Reply by wiiiimm** [reply]
@SSardorf can you please verify that your Notion credentials are configured properly as ENVs and your app can see these ENVs after deployment?

**Reply by SSardorf** [reply]
Yes, I'm still having these issues. And indeed, I have deployed the env variables correctly. That works in logging
